.storybook/webpack.config.js

```diff
@@ -1,30 +1,48 @@
-const env = require("dotenv")
-const fs = require("fs")
-const path = require("path")
-const sharify = require("./sharify")
+const env = require("dotenv");
+const fs = require("fs");
+const path = require("path");
+const sharify = require("./sharify");
+
+const webpack = require("webpack");
+const webpackMerge = require("webpack-merge");
+const { CheckerPlugin } = require("awesome-typescript-loader");
 
 /**
  * Write out a file that stubs the data that’s normally shared with the client through the `sharify` module. This file
  * is then replaced in the product of webpack where normally the actual `sharify` module would be loaded.
  */
-const { METAPHYSICS_ENDPOINT } = env.config().parsed
-const sharifyPath = sharify({ METAPHYSICS_ENDPOINT })
+const { METAPHYSICS_ENDPOINT } = env.config().parsed;
+const sharifyPath = sharify({ METAPHYSICS_ENDPOINT });
+
+// A mix of  the base from Emission's webpack setup, and the simple config for
+// storybooks: https://storybook.js.org/configurations/custom-webpack-config/
 
 module.exports = {
   devtool: "#inline-source-map", // Otherwise getting errors about e.g. `Relay` not being defined.
   resolve: {
-    extensions: ["", ".js", ".jsx", ".ts", ".tsx"],
+    extensions: [".js", ".jsx", ".ts", ".tsx"],
```

Comment:
Webpack 2 errors on load for that specifically, think whatever that implied is now handled implicitly